website/docs/registry/providers/publishing.html.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: "registry"
+page_title: "Terraform Registry - Publishing Providers"
+sidebar_current: "docs-registry-provider-publishing"
+description: |-
+  Publishing Providers to the Terraform Registry
+---
+
+~> __Publishing Beta__<br>
+~> Welcome! Thanks for your interest participating in our Providers in the Registry beta! Paired with Terraform 0.13, our vision is to make it easier than ever to discover, distribute, and maintain your provider(s). We welcome any feedback you have throughout the process and encourage you to reach out if you have any questions or issues by emailing terraform-registry-beta@hashicorp.com.
```

Comment:
I'm not 100% sure here but the `~>` extension in our markdown renderer is kinda finicky due to how it's implemented, and I worry that this might produce two separate callout boxes, rather than one with a newline in it. I've found that in order for these to work as expected the content needs to be written all as one big Markdown paragraph with a leading `~>`.

I've not tried putting a `<br>` in like this before so this is kinda uncharted and if you've seen it work in preview then please ignore this! But if this ends up rendering oddly, I'd suggest trying to reformat it all on one line like this:

```
~> __Publishing Beta__<br> Welcome! Thanks for your interest...
```

More subjectively, this feels more like a note (`->`) than a warning (`~>`) to me. From a practical standpoint, the only real difference is that a note is rendered in a blue callout box, while a warning is in a yellow one. I could see it being a warning if the intent was to say "Warning! This is only in beta!", but the current phrasing seems more informational, like "By the way, here's how to contact us if you have feedback."

Reply:
Updated! I agree on the note vs warning.

Reply:
<img width="894" alt="Screen Shot 2020-06-25 at 3 26 11 PM" src="https://user-images.githubusercontent.com/246978/85786448-442a7800-b6f8-11ea-9402-a59335913280.png">